Makefile

```diff
@@ -58,6 +58,11 @@ build:
 	GOOS=${GOOS} GOARCH=${GOARCH} ${PROXY} go build -ldflags "$(LDFLAGS)" -v -o bin/containerd-nydus-grpc ./cmd/containerd-nydus-grpc
 	GOOS=${GOOS} GOARCH=${GOARCH} ${PROXY} go build -ldflags "$(LDFLAGS)" -v -o bin/nydus-overlayfs ./cmd/nydus-overlayfs
 
+.PHONY: static
```

Comment:
Because we do not know what is going inside the kind control plane where the binary will get copied, we do not a static build. There is already a 'release-static' target, but it does require rust / the optimization thing.

Reply:
Thanks, how about splitting `static-release` into `static` and `static-optimizer`? but the current changes is also LGTM.

Reply:
Yep, cleaning-up the Makefile in that context would definitely make sense.